pkg/controller/disruption/disruption_test.go

```diff
@@ -146,7 +145,7 @@ var customGVK = schema.GroupVersionKind{
 	Kind:    "customresource",
 }
 
-func newFakeDisruptionController() (*disruptionController, *pdbStates) {
+func newFakeDisruptionController(ctx context.Context) (*disruptionController, *pdbStates) {
```

Comment:
I'm sorry for nit-picking, but can we maybe switch that to be more unified with what we're doing in other tests?

- there is a function that just creates the controller, e.g.:
https://github.com/kubernetes/kubernetes/blob/master/test/integration/replicaset/replicaset_test.go#L120
- there is a function that actually starts that, e.g. https://github.com/kubernetes/kubernetes/blob/master/test/integration/replicaset/replicaset_test.go#L155
- that function returns destroyFn which is explicitly deferred in the test, e.g.:
https://github.com/kubernetes/kubernetes/blob/master/test/integration/replicaset/replicaset_test.go#L502

I know that your code is technically correct but we would benefit from more consistency across our codebase...

Reply:
Actually sorry - I realized that those are unit tests, so maybe leaving it here as is is fine for now to stop the bleeding...